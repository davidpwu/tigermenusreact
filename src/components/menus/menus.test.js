import React from "react";
import {shallow} from "enzyme";
import Menus from "./menus.component";

test("render Menus component", () => {
  expect(shallow(<Menus />)).toMatchSnapshot();
});
