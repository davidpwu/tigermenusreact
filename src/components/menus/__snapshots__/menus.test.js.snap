// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Menus component 1`] = `ShallowWrapper {}`;
